end2end/cypress/integration/ObjectTypes.spec.js

```diff
@@ -0,0 +1,122 @@
+var thisId
```

Comment:
Good idea! I added the following comment.
// Declare a global variable to store the id of a newly created object type so that we can refer to it in subsequent tests